client/src/components/Connections/scenes/Results/Results.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react'
+import ConnectionWrapper from '../../dataWrappers/ConnectionWrapper'
+import EntityWrapper from '../../dataWrappers/EntityWrapper'
+import InfoLoader from './components/InfoLoader/InfoLoader'
+
+const Results = (props) => (
+  <div>
+    {props.connections && props.connections.length > 0 ? (
```

Comment:
when you have components relying on something existing in state, you can leverage data-providers, which do not render anything until they have

if you need to display some component if you have a part of state setup and display something else if not, you can also do it with recompose and branch - https://github.com/acdlite/recompose/blob/master/docs/API.md#branch

Reply:
Rewritten.